browser/js/app-initializer.js

```diff
@@ -27,6 +28,7 @@ export const presets = {
 		ads: true,
 		syndication: true,
 		roe: true,
+		feedback: true,
```

Comment:
Is it safe to make it on by default yet? Yes we should probably have it default to on when we're happy with the UI, but are we ok with doing that right now?